server/app/repository/QuestionRepository.java

```diff
@@ -173,6 +175,17 @@ public ImmutableList<QuestionDefinition> getAllQuestionsForTag(QuestionTag tag)
         .collect(ImmutableList.toImmutableList());
   }
 
+  public ImmutableSet<String> getQuestionNames() {
+    ImmutableSet.Builder<String> names = ImmutableSet.builder();
+    List<SqlRow> rows = database.sqlQuery("SELECT DISTINCT name FROM questions").findList();
```

Comment:
Any specific reason to use raw SQL rather than the ORM?

Reply:
I wasn't able to figure out how to do it with the ORM in a way that didn't fetch all columns of the table